changelogs/current.yaml

```diff
@@ -107,6 +107,9 @@ bug_fixes:
 - area: jwt_authn
   change: |
     fix a bug that jwt_cache breaks the :ref:`provider_and_audiences <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtRequirement.provider_and_audiences>` JWT requirement.
+- area: health_checker
+  change: |
+    prevent writing pending data for health checkers by introducing ``ConnectionCloseType::Abort``.
```

Comment:
For my understanding, why does this matter? What would happen previously? Should we include some detail in the release note?

Reply:
The detail is in #23718. 
tl;dr there's cascading cpu backpressue for health checker handshake while cpu is throttled by docker cgroup limit. More and more HC requests are timeout-ed and closed before handshake, but we still try to TLS handshake.

IMO runtime guard is **not** needed for now since it doesn't matter to give up health checker's data.

However it needs some deliberation to replace as much `NoFlush` with `Abort` as possible in the future
https://github.com/envoyproxy/envoy/issues/23718#issuecomment-1341338352

I'll mention the detail in the release note in next commit.